Lib/idlelib/config.py

```diff
@@ -778,9 +779,99 @@ def _warn(msg, *key):
         _warned.add(key)
 
 
+class ConfigChanges(dict):
+    """Manage a user's proposed configuration option changes.
+
+    Names used across multiple methods:
+        page -- one of the 4 top-level dicts representing a
+                .idlerc/config-x.cfg file.
+        config_type -- name of a page.
+        section -- a section within a page/file.
+        item -- name of a value within a section.
+        value -- value for the item.
+
+    Methods
+        add_item:
```

Comment:
Would you prefer to use `add_item` in above changes? Current code is using `additem`.